makefile

```diff
@@ -35,22 +35,25 @@ endif
 
 
 # start evergreen specific configuration
-unixPlatforms ?= linux_amd64 darwin_amd64 $(if $(STAGING_ONLY),,linux_386 linux_s390x linux_arm64 linux_ppc64le linux_amd64_legacy freebsd_amd64)
-windowsPlatforms ?= windows_amd64 $(if $(STAGING_ONLY),,windows_386)
+
+unixPlatforms := linux_amd64 darwin_amd64 $(if $(STAGING_ONLY),,linux_386 linux_s390x linux_arm64 linux_ppc64le linux_amd64_legacy freebsd_amd64)
```

Comment:
I'd kind of prefer consistency here, since I suspect that people will see `:=` and `?=` and not really understand why it's one way in one place and another in another. Could we use `:=` everywhere?